BuildingSync.xsd

```diff
@@ -17787,4 +17790,468 @@
+      <xs:element ref="auc:UserDefinedFields" minOccurs="0"/>
+    </xs:sequence>
+    <xs:attribute name="ID" type="xs:ID" use="required"/>
+  </xs:complexType>
+  <xs:simpleType name="IntervalFrequencyType">
+    <xs:annotation>
+      <xs:documentation>Indicates frequency of data that's available for a given variable. Data that's available can range from 10 minute interval to annual. This interval frequency can be applied to resource or other time series data like weather.</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="1 minute"/>
+      <xs:enumeration value="10 minute"/>
+      <xs:enumeration value="15 minute"/>
+      <xs:enumeration value="30 minute"/>
+      <xs:enumeration value="Hour"/>
+      <xs:enumeration value="Day"/>
+      <xs:enumeration value="Week"/>
+      <xs:enumeration value="Month"/>
+      <xs:enumeration value="Annual"/>
+      <xs:enumeration value="Quarter"/>
+      <xs:enumeration value="Other"/>
+      <xs:enumeration value="Unknown"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:simpleType name="UnitsType">
+    <xs:annotation>
+      <xs:documentation>Enumeration for different potential units.</xs:documentation>
+    </xs:annotation>
+    <xs:union memberTypes="auc:ResourceUnitsType auc:PressureUnitsType auc:PeakResourceUnitsType auc:TemperatureUnitsType"/>
+  </xs:simpleType>
+  <xs:simpleType name="DimensionlessUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:DimensionlessUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="PeakResourceUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:PeakResourceUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="PressureUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:PressureUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="ResourceUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:ResourceUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="TemperatureUnitsType">
+    <xs:union memberTypes="auc:OtherUnitsType auc:TemperatureUnitsBaseType"/>
+  </xs:simpleType>
+  <xs:simpleType name="OtherUnitsType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Other"/>
+      <xs:enumeration value="Unknown"/>
+      <xs:enumeration value="None"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:simpleType name="DimensionlessUnitsBaseType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Percent, %"/>
+      <xs:enumeration value="Percent Relative Humidity, %RH"/>
+      <xs:enumeration value="Percent, %"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:simpleType name="PeakResourceUnitsBaseType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="kW"/>
+      <xs:enumeration value="MMBtu/day"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:simpleType name="PressureUnitsBaseType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Pascal, Pa"/>
```

Comment:
I think the majority of BulidngSync is in USCS units, so kPa and Pa would not show up here, unfortunately.

Reply:
fixed